lib/request_headers_logger/text_formatter.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module RequestHeadersLogger
+  class TextFormatter < ::Logger::Formatter
+    def call(severity, timestamp, progname, msg)
+      super(severity, timestamp, progname, "#{tags_text}#{msg}")
+    end
+
+    def tags_text
+      RequestHeadersLogger.tags.collect { |_key, tag| "[#{tag}] " }.join if RequestHeadersLogger.tags.any?
```

Comment:
Same here but less problematic. Implicit dep the ActiveSupport::TaggedLogger.

Reply:
[`RequestHeadersLogger.tags`](https://github.com/fidor/request_headers_logger/blob/master/lib/request_headers_logger.rb#L14)  is just the store items after filtering the whitelisted ones.